mcp300x_block.py

```diff
@@ -61,6 +61,7 @@ class MCP300x(EnrichSignals, Block):
 
     version = VersionProperty('0.1.0')
     channel = IntProperty(default=0, title="Channel Number")
+    total_channels = IntProperty(default=8, title="Total Channels")
```

Comment:
Looks like this was added to the README but then got removed after fixing a merge conflict. Please add this to the README and spec files.